torch/csrc/jit/passes/quantization.cpp

```diff
@@ -140,6 +144,17 @@ std::vector<std::string> _single_input_general_aten_funcs = {
     "tanh",
 };
 
+// Theses are prim::CallFunctions for ops that doesn't require observation and
+// have a single input Tensor
+// Also these ops do computation on the value of Tensor
+std::vector<std::string> _single_input_general_value_call_funcs = {};
+
+// Theses are aten functions for ops that doesn't require observation and
+// have a single input Tensor
+// Also these ops do computation on the value of Tensor
+// e.g. `aten::maxpool(%input_tensor, ...)`
```

Comment:
Comment says maxpool as an example but we have max_pool ops in `_single_input_general_shape_aten_funcs`

Reply:
yeah, I'll move these op later here later